misc/python3/depgraph.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -x
+set -eo pipefail
+
+if [ "${CIRCLE_BRANCH}" != "master" ]; then
+    echo "Declining to update dependency graph for non-master build."
+    exit 0
+fi
+
+TAHOE="${PWD}"
+git clone -b gh-pages git@github.com:tahoe-lafs/tahoe-depgraph.git
+cd tahoe-depgraph
+
+# Generate the maybe-changed data.
+python tahoe-depgraph.py "${TAHOE}"
+
+if git diff-index --quiet HEAD; then
+  echo "Declining to commit without any changes."
+  exit 0
+fi
+
+# Commit everything that changed.  It should be tahoe-deps.json and
+# tahoe-ported.json.
+git config user.name 'Build Automation'
```

Comment:
can/should we just hard-code those names in the commit then..?

Reply:
I guess that would be `git add tahoe-deps.json tahoe-ported.json` and then `git commit -m` instead of `git commit -am`?